src/pages/SocietyExec.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Container, Grid, Typography, Paper } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import Avatar from '../components/Avatar';
+import { hostname} from '../links';
+
+
+
+const useStyles = makeStyles(theme => ({
+  root: theme.root,
+  container: theme.page,
+  paper: {
+    ...theme.paper,
+    padding: theme.spacing(4),
+  },
+  link: {
+    ...theme.link,
+    float: 'right',
+    textDecoration: 'none',
+  },
+  carousel: {
+    margin: 'auto',
+    paddingTop: theme.spacing(4),
+  },
+}));
+
+export default function SocietyExec({sid}) {
+  const classes = useStyles();
+
+  const [member, setMember] = useState([]);
+
+  useEffect(() => {
```

Comment:
This functionality is being developed by @sannidhi-s-shetty, please coordinte with her.